<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>521</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Sayı Tahmin Etme Oyunu</string>
   </property>
   <widget class="QLineEdit" name="lineEditNoun1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>113</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNoun2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>113</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Aralık</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Aralık</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuessInsertNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Edilecek Sayıyı Üret</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuess">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Et</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
